docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/ra1n-chromeos.md

```diff
@@ -0,0 +1,80 @@
+---
+lang: en_US
+title: Installing Odysseyra1n (ChromeOS)
+description: Guide to installing Odysseyra1n on ChromeOS
+permalink: /installing-odysseyra1n-chromeos
+redirect_from:
+  - /installing-checkra1n/chromeos
+  - /installing-checkra1n-chromeos
+extra_contributors:
+  - DhinakG
+  - Tanbeer191
+  - TheHacker894
+  - zachary7829
+  - stekc
+  - WhitetailAni
+---
+
+::: warning
+
+If you are using a computer with an AMD Ryzen CPU, you will likely run into issues. If you do run into issues, you should use a Mac or a computer with an Intel CPU to follow this guide.
+
+:::
+
+::: warning
+
+If you have an Apple TV HD and are using a USB-C to UsB-C to do this process, you may run into issues entering into DFU mode
+
+If you do have issues, get a USB-A to USB-C cable and, if necessary, also get a USB-C to USB-A adapter.
+
+:::
+
+::: danger
+
+This is a highly experimental guide page, there may be things not directly mentioned that need to be done in order to follow this. As a result, we highly recommend getting another type of computer instead if possible.
+
+:::
+
+::: tip
+
+You need to have enabled developer mode on your ChromeOS device to follow this
+
+:::
+
+## Requirements (ChromeOS)
+
+- The latest release of [checkra1n](https://checkra.in)
+- An Apple TV Siri remote (any generation)
+- If you have an Apple TV 4K, you will also need two cables
+  - A GoldenEye, or Foxlink X892 cable
+  - An Alex DCSD cable
+It is recommended to purchase these from AliExpress for the best prices. Note that they typically take about a month to arrive.
+
+![A screenshot of the checkra1n application](/assets/images/checkra1n.png)
+
+## Allowing checkra1n to run
+
+1. Move the checkra1n binary which is downloaded to `/usr/local/bin`
+    - This can be done by doing the following:
+      - press `Ctrl` + `Alt` + `T`
+      - run `shell`
+      - run `mv ~/Downloads/checkra1n /usr/local/bin/`
```

Comment:
Please capitalise the first letters of each point. This should be changed on original chromeos guide if it's there too